flink-runtime/src/test/java/org/apache/flink/runtime/clusterframework/types/ResourceProfileTest.java

```diff
@@ -18,19 +18,21 @@
 
 package org.apache.flink.runtime.clusterframework.types;
 
+import org.apache.flink.api.common.operators.ResourceSpec;
 import org.junit.Test;
 
+import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
 public class ResourceProfileTest {
 
 	@Test
 	public void testMatchRequirement() throws Exception {
-		ResourceProfile rp1 = new ResourceProfile(1.0, 100, 100, 100);
-		ResourceProfile rp2 = new ResourceProfile(1.0, 200, 200, 200);
-		ResourceProfile rp3 = new ResourceProfile(2.0, 100, 100, 100);
-		ResourceProfile rp4 = new ResourceProfile(2.0, 200, 200, 200);
+		ResourceProfile rp1 = new ResourceProfile(1.0, 100, 100, 100, 0, 0, null);
```

Comment:
better to use `Collections.emptyMap` because the extended resources is not marked as `Nullable`.